octoprint_mrbeam/camera/label_printer.py

```diff
@@ -221,6 +214,8 @@ def _get_model_abbrev(self):
             model = "DCR"
         elif self._device_info.get_model() in (self.MRBEAM_2_DC):
             model = "DC"
+        elif self._device_info.get_model() == self.MRBEAM_2_DC_S:
```

Comment:
```suggestion
        elif self._device_info.get_model() in self.MRBEAM_2_DC_S:
```
just for consistency

Reply:
so you mean `X in (self.MRBEAM_2_DC_S,):` ? It is slightly less readable.

The way you wrote it only works because strings also have a weird list comprehension, as in
```python
>>> "a" in "aab"
True
>>> "aab" in "aab"
True
>>> "aabb" in "aab"
False
```
This is also why it works for line 215. If it was integers, this wouldn't work.